Revise elec demand

uvchik opened the pull request.

uvchik committed:
Add bmwi-tests

uvchik committed:
Fix entsoe test

uvchik committed:
Add version parameter to download older opsd versions

uvchik committed:
Keep sectors in openego

uvchik committed:
Add slp calculation and fix timestamp

uvchik committed:
Fix docs

uvchik committed:
Fix time index with freq

uvchik committed:
Add germany polgon with or without AWZ

pep8speaks commented:
Hello @uvchik! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:







* In the file [`reegis/entsoe.py`](https://github.com/reegis/reegis/blob/a80953e30ada8a214d96f98740471d176a6ab64f/reegis/entsoe.py):

> [Line 248:5](https://github.com/reegis/reegis/blob/a80953e30ada8a214d96f98740471d176a6ab64f/reegis/entsoe.py#L248): [E731](https://duckduckgo.com/?q=pep8%20E731) do not assign a lambda expression, use a def

stickler-ci committed:
Fixing style errors.

uvchik committed:
Move lambda function directly to parser

uvchik committed:
Use latest (unreleased) version of the demandlib

uvchik committed:
Fix tests

uvchik committed:
Add minimal python version

uvchik committed:
Fix strange test

I do not understand, why the test failed, because it does not fail on
the local machine.

stickler-ci committed:
Fixing style errors.

uvchik committed:
Fix style issues

uvchik committed:
Use the latest version of the windpowerlib

uvchik committed:
Fix setup
